src/icon-manifest/supportedFolders.ts

```diff
@@ -1,33 +1,31 @@
 /* tslint:disable max-line-length */
-
 import { IExtensionCollection, IFolderExtension, FileFormat } from '../models/IExtension';
 
 export const extensions: IExtensionCollection<IFolderExtension> = {
   supported: [
-    { icon: 'aws', extensions: ['aws'], format: FileFormat.svg },
-    { icon: 'aws', extensions: ['aws'], dot: true, format: FileFormat.svg },
+    { icon: 'aws', extensions: ['aws', '.aws'], format: FileFormat.svg },
```

Comment:
I used `svg` deliberately, yes, but I've added a link to the supported extensions (as per your suggestion).

I'm not sure what do you mean by default extensions here...